.cirrus.yml

```diff
@@ -325,7 +325,7 @@ task:
 # WINDOWS SHARDS
 task:
   windows_container:
-    image: cirrusci/android-sdk:28-windowsservercore-2019
+    image: cirrusci/android-sdk:30-windowsservercore-2019
```

Comment:
@fkorotkov we're ready when you are :)

Reply:
what do you mean? 😅 seems the image is working for you 🤔

Reply:
ah ya LG. You were just mentioning on chat previously whether we needed 30. Awesome, thanks!

Reply:
oh, right. I was asking in the context of using 30 in the `catalina-xcode-12.0-flutter`. I'll upgrade it once this PR merged or it doesn't matter and I can bump it any time?

Reply:
You just meant creating a new catalina-xcode-12.0-flutter rather than moving all our current tests to catalina-xcode-12.0-flutter right? if so then ya, any time is good.

Reply:
So the question here is do you want Android 30 as part of this PR #63252? cc @jmagman

Reply:
If I understand your question, it's 2 separate things. This is Android 30 for Windows, that one is Xcode 12 for macOS. We can do them separately.